input/gcamdata/inst/extdata/gcam-india/SE_PopH_Census_1961_2011.csv

```diff
@@ -0,0 +1,43 @@
+# File: SE_PopH_Census_1961_2011.csv,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
Why are there two historical population files?

Reply:
We have prepared another set of input files with some updates:
1. We have grouped all the union territories under one head.
2. We have changed some assumptions to estimate the data of the newly formed state about which we were discussing in the last call.
3. We have standardized our structure of naming the files so that it could be easy for us to understand the file from its title. The new naming format which we are following now is:
Module_Sector_Subsector_Data_Source_Years
eg: Here in SE_PopH_Census_1961_2011, SE is Socioeconomics and PopH is population historical from 1961 to 2011.